src/main/resources/templates/repo/agent/bedrock_agent_template.json

```diff
@@ -104,13 +104,23 @@
 									}
 								},
 								{
-									"Description": "Use this function to discover the list of children that belong to a container entity such as a project or folder.",
+									"Description": "Use this function to discover the entire list of children that belong to a container entity such as a project or folder. Loop until the response has nextPageToken parameter to retrieve all the results.",
```

Comment:
Use this function to discover the list of children that belong to the container entity.  The results are paginated.